l10n_it_appointment_code/readme/INSTALL.rst

```diff
@@ -1,17 +1,17 @@
 **Italiano**
 
-Qualora questo modulo venisse installato in un DB dove ``l10n_it_codici_carica`` è già installato, è necessario eseguire la sequente procedura:
```

Comment:
Grazie per il typo :eyeglasses: 
non vanno bene i due punti? per favore, a prescindere dalla decisione che vorrai prendere, uniforma la scelta anche sulla parte inglese.

Reply:
> Grazie per il typo eyeglasses
> non vanno bene i due punti? per favore, a prescindere dalla decisione che vorrai prendere, uniforma la scelta anche sulla parte inglese.

L'avevo lasciata così perché in inglese, in particolare americano, utilizzano anche la maiuscola dopo i due punti (così come fa Odoo stesso).
Per me no problem comunque, la uniformo all'italiano.